spec/support/setup_azure_blob_containers.rb

```diff
@@ -10,9 +10,13 @@
       client.create_container(container_name) unless containers.map(&:name).include?(container_name)
 
       # Empty container
-      client.list_blobs(container_name).each do |blob|
-        client.delete_blob container_name, blob.name
+      puts "Emptying container #{container_name}"
```

Comment:
Part of finding out why test suite was slow.  Decided to keep this in as it is useful